### PR TITLE
ngsplot

### PR DESCRIPTION
**Pull request by orbsmiv**
For INC1075861

`ngsplot-2.63-foss-2019b-Python-2.7.16-R-3.6.2.eb`
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/n/ngsplot/ngsplot-2.63-foss-2019b-Python-2.7.16-R-3.6.2.eb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+]
+
+dependencies = [
+    ('Python', '2.7.16'),
+    ('R', '3.6.2'),
+    ('R-bundle-Bioconductor', '3.10', '-R-%(rver)s'),
+]
+
+# This builds a distributable tar file, which we want to unpack...
+buildopts = " && tar xf ngsplot-2.63.tar.gz"
```
Comment (branfosj) [inline easybuild/easyconfigs/n/ngsplot/ngsplot-2.63-foss-2019b-Python-2.7.16-R-3.6.2.eb:29]: I feel a simplification here. If the ngsplot Makefile creates this tar.gz from existing items then we can just patch the Makefile to skip this part.

**Review by branfosj — CHANGES_REQUESTED** [review]
see comment on build process

File: easybuild/easyconfigs/n/ngsplot/ngsplot-2.63-foss-2019b-Python-2.7.16-R-3.6.2.eb (hunk 1 above)
Comment (orbsmiv) [reply easybuild/easyconfigs/n/ngsplot/ngsplot-2.63-foss-2019b-Python-2.7.16-R-3.6.2.eb:29]: Okay, I'll have a look – to be honest I pulled the 2017a version, made the necessary toolchain adjustments and hoped for the best!

Comment (orbsmiv) [reply easybuild/easyconfigs/n/ngsplot/ngsplot-2.63-foss-2019b-Python-2.7.16-R-3.6.2.eb]: Resolved with b9b290d

**Review by branfosj — APPROVED** [review]
👍

**Pull request merged**